ToggleButton/ToggleButton.module.less

```diff
@@ -62,22 +62,81 @@
 
 .button {
 	.applySkins({
+		min-width: @agate-togglebutton-min-width;
```

Comment:
It looks fine to me because it overrides `min-width: @agate-button-height` in Button.module.less and `@agate-button-height` is equal to `@agate-togglebutton-min-width` in all skins except `silicon`.
I just wrote the comment because we could be curious about this code later.